### PR TITLE
Create EWMS_Riedel_Startup.yaml

### PR DESCRIPTION
**Pull request by rachellombardi**

### CODE REVIEW

**Review by matyasselmeci — APPROVED** [review]

File: projects/EWMS_Riedel_Startup.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Department: Physics
+Description: To develop an Observation Management System Framework which will help alleviating multiple types of astrophysical data and respective workflow management. The Event Workflow Management System (EWMS)is a workload manager designed for processing events (simulated readouts from a particle physics detector, recorded data points, images, etc.) with complex workflows.
+FieldOfScience: Astrophysics
+Organization: Event Workflow Management System (EWMS)
+PIName: Benedikt Riedel
+Sponsor:
+  VirtualOrganization:
```
Comment (matyasselmeci) [inline projects/EWMS_Riedel_Startup.yaml:7]:
```suggestion
  CampusGrid:
```

File: projects/EWMS_Riedel_Startup.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Department: Physics
+Description: To develop an Observation Management System Framework which will help alleviating multiple types of astrophysical data and respective workflow management. The Event Workflow Management System (EWMS)is a workload manager designed for processing events (simulated readouts from a particle physics detector, recorded data points, images, etc.) with complex workflows.
+FieldOfScience: Astrophysics
+Organization: Event Workflow Management System (EWMS)
```
Comment (matyasselmeci) [inline projects/EWMS_Riedel_Startup.yaml:4]: Is EWMS really an organization? Shouldn't it be "University of Wisconsin-Madison" or something?

Comment (rachellombardi) [inline projects/EWMS_Riedel_Startup.yaml:4]:
```suggestion
Organization: University of Wisconsin-Madison
```

Comment (rachellombardi) [reply projects/EWMS_Riedel_Startup.yaml:4]: A google search of the grant shows that it went to Madison, so I think you're correct.

**Pull request merged**